templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
```

Comment:
Not sure about backend ports what is that referring to?  

In any case, if we set up the ELB in CFn to go directly to the master, we can have it map 443 to 6443 on the master.  No need to muck with ports.

The thing is -- I'm not sure we can set up the ELB before the master.  We can't create circular dependencies here or I'm sure CFn will kick it back at us.